apps/recnet/src/app/admin/article/management/page.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+
+import { useState } from "react";
+import { cn } from "@recnet/recnet-web/utils/cn";
+import { Button, Flex, TextField } from "@radix-ui/themes";
+import { toast } from "sonner";
+
+import { AdminSectionBox, AdminSectionTitle } from "../../AdminSections";
+
+import { ArticleManagementForm } from "./ArticleManagementForm";
+
+import { trpc } from "@recnet/recnet-web/app/_trpc/client";
+
+export default function ArticleManagementPage() {
+  const [searchUrl, setSearchUrl] = useState("");
+  const [foundArticle, setFoundArticle] = useState<any>(null);

```

Comment:
```suggestion
  const [foundArticle, setFoundArticle] = useState<Article | null>(null);

```

You should use the type from `@recnet/recnet-api-model`. You can add a line on top of the file `import { Article } from "@recnet/recnet-api-model"`